app/views/home/about.html.erb

```diff
@@ -104,6 +105,12 @@
 
         <br />
 
+        <h2>What about submissions that rely upon proprietary languages/development environments?</h2>
+
+        <p>We strongly prefer software that doesn't rely upon proprietary (paid for) development environments/programming languages. However, provided <em>your</em> submission has meets our submission requirements (including having a valid open source license) then we will consider your submission for review. Should your submission be accepted for review, we may ask you, the submitting author to help us find reviewers who already have the required development environment installed.</p>
```

Comment:
`you, the submitting author` -> either just `you` or `you, the submitting author, `